src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,175 @@
+          padding: '1em',
+          boxSizing: 'border-box',
+          border: 'none',
+          boxShadow: '1px 0px 5px #efefef',
+          transition: '.5s all ease',
+        },
+      }}
+      variant="persistent"
+      anchor="left"
+      open={isMobile ? isOpen : true}
+    >
+      {children}
+    </Drawer>
+  );
+};
+
+// Sidebar Header
+const DrawerHeader = ({ handleDrawerClose, isOpen }: { handleDrawerClose: () => void, isOpen: boolean }) => {
+  return (
+    <>
+      <div style={{
+        display: 'grid',
+        gridTemplateColumns: isOpen ? '1fr 1fr 1fr' : '1fr'
+      }}>
+        {
+          isOpen && (
+            <Fade in={isOpen}>
+              <img
+                alt="WBD Logo"
+                src="logo.png"
+                style={{
+                  height: 40,
+                  gridColumn: '2',
+                  justifySelf: 'center',
+                }}
+              />
+            </Fade>
+          )
+        }
+
+        <IconButton
+          aria-label='close/toggle drawer'
+          onClick={handleDrawerClose}
+          sx={{
+            color: 'primary.main',
+            justifySelf: isOpen ? 'end' : 'center',
+            rotate: isOpen ? '0deg' : '180deg',
+            transition: '.5s all ease',
+          }}>
+          <ChevronLeftIcon />
+        </IconButton>
+      </div>
+      <Divider sx={{
+        backgroundColor: 'primary.main',
+        marginY: '1em',
+        height: '2px'
+      }}
+      />
+    </>
+  );
+};
+
+// Menu
+const DrawerMenu = () => {
+  const menuItems = [
+    {
+      text: 'Página Inicial',
+      icon: <Home />,
+    },
+    {
+      text: 'Cadastro',
+      icon: <AssignmentAdd />,
+    },
+    {
+      text: 'Alunos',
+      icon: <PeopleAlt />,
+    },
+    {
+      text: 'Frequência',
+      icon: <Checklist />,
+    },
+    {
+      text: 'Atividades',
+      icon: <AppRegistration />,
+    },
+  ];
+
+  return (
+    <List
+      sx={{
+        color: 'primary.main',
+      }}
+    >
+      {menuItems.map((item) => (
+        <ListItem key={item.text} disablePadding>
+          <ListItemButton sx={{
+            backgroundColor: '#efefef',
+            borderRadius: '.7em',
+            padding: '.7em 1em',
+            justifyContent: 'center',
+            alignItems: 'center',
+            display: 'flex',
+            marginBottom: '1em',
+          }} aria-label='menu item'>
+            <ListItemIcon sx={{ color: 'primary.main', minWidth: 'fit-content' }}>
+              {item.icon}
+            </ListItemIcon>
+            <ListItemText
+              primary={item.text}
+              sx={{ '& .MuiListItemText-primary': { fontWeight: 'bold', textWrap: 'nowrap', overflow: 'hidden', marginLeft: '1em' } }}
+            />
+          </ListItemButton>
+        </ListItem>
+      ))}
+    </List>
+  );
+};
+
+export const Sidebar = () => {
+  const [open, setOpen] = useState(false);
+  const isMobile = useMediaQuery('(max-width: 600px)');
```

Comment:
We can be more responsive using Mui's theme object
```suggestion
  const theme = useTheme();
  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
```